x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/DownsampleStep.java

```diff
@@ -93,11 +86,9 @@ public void performAction(
                 );
                 listener.onResponse(null);
             }
```

Comment:
If the index doesn't have the SUCCESS tag but we're re-executing the downsample ILM step we should call `performDownsampleIndex(indexName, downsampleIndexName, ActionListener.wrap(listener::onResponse, listener::onFailure));` to start waiting for a result or get an error message from the transport action